R/copy-model-from.R

```diff
@@ -2,7 +2,10 @@
 # Iterating on models
 #######################
 
-#' Generic S3 method from iterating on models.
+#' Create new model by copying existing model
+#'
+#' Create new model by copying existing model. Useful for iterating during model development.
+#' Also fills `based_on` field by default, for constructing model ancestry. See \href{../articles/using-based-on.html}{"Using based_on field" vignette} for details.
```

Comment:
I think we need to [enable markdown support](https://roxygen2.r-lib.org/articles/markdown.html#turning-on-markdown-support) in `roxygen` comments for `based_on` and similar to render correctly.